Firestore/Source/Public/FIRFirestore.h

```diff
@@ -110,6 +110,10 @@ NS_SWIFT_NAME(Firestore)
  * Executes the given updateBlock and then attempts to commit the changes applied within an atomic
  * transaction.
  *
+ * The maximum number of writes allowed in a single transaction is 500, but note that each usage of
+ * `FieldValue.serverTimestamp() `FieldValue.arrayUnion()`, `FieldValue.arrayRemove()`, or
```

Comment:
missing a comma

Reply:
Also, unbalanced backticks, here and throughout.